pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -211,6 +215,39 @@ func chooseRandomNode(windowsVMs map[types.WindowsVM]bool) types.WindowsVM {
 	return nil
 }
 
+// deleteWindowsVM terminates the VM with the given id through the cloud provider
```

Comment:
Doc name doesn`t match with the function name

Reply:
We now have `destroy`, `delete` and `terminate` when it comes to deleting VMs. Let us consolidate on a single term. Moreover I don't see the point of the helper.